src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const spans = document.querySelectorAll('.population');
+
+const populations = [];
+
+spans.forEach(span => {
+  populations.push(span.textContent);
+});
+
+const populationValue = [];
+
+populations.forEach(population => {
+  populationValue.push(parseInt(population.replace(/,/g, '')));
+});
+
+const totalPopulation = populationValue.reduce((a, b) => a + b, 0);
```

Comment:
```suggestion
const totalPopulation = [...spans].reduce((acc, item) => acc + parseInt(item.textContent.replace(/,/g, '')), 0);
```

you can make all it in reduce.
two forEach iterations redundant